vagrant/roles/local.defaults/vars/main.yml

```diff
@@ -38,7 +38,7 @@ vagrant:
     private: 192.168.122.0/24
     public: 192.168.123.0/24
 
-  # Storage pool to use for the virtual disks (optional)
+#  # Storage pool to use for the virtual disks (optional)
```

Comment:
extra hash(`#`)?

Reply:
No. This one is on purpose. The problem here is that comments in yaml are required to be put at the same indentation as the next line. This comment is related to the setting in the next line, so it's "re-commented" because the setting is also commented. The first hash is required to avoid a yamllint error. The second hash is left there so that if you uncomment both lines by removing the first hash, the comment will be correctly indented without requiring manual indentation.

I think this is the cleanest/easier way to avoid yamllint errors for these comments. The alternative would be to disable comment rule in `.yamllint` and allow any indentation for comments.

Reply:
Got it, thanks for the explanation.